src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayCompilerInternalOptions.cs

```diff
@@ -54,5 +54,17 @@ internal enum SymbolDisplayCompilerInternalOptions
         /// Append '!' to non-nullable reference types.
         /// </summary>
         IncludeNonNullableTypeModifier = 1 << 6,
+
+        /// <summary>
+        /// Append '?' to nullable reference types.
+        /// Cannot be used in conjunction with <see cref="IncludeNonNullableTypeModifier"/>.
```

Comment:
>Cannot be used in conjunction with <see cref="IncludeNonNullableTypeModifier"/>. [](start = 12, length = 80)

It doesn't look like this comment is in agreement with a comment and an assert in SymbolDisplayVisitor.Types.cs (line 27). Also, Rather than asserting consistency of options in the visitor, consider asserting it when SymbolDisplayFormat instance is created. This way, the assert will provide more help in finding the source of the inconsistency.